build/config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "locales": [
+    "en"
+  ],
+  "outDir": "api\\worker\\WorkerComm",
```

Comment:
Why this specific file is hardcoded here?

Reply:
This line isn't hardcoded. The script actually modifies the config and jsdoc file to change the input and output directory path for every js file when converting them to mdx. So, this was the last file's output directory path. 

Will update the script to set some default value to the outDir and inputDir after all the processing is done.